config/locales/en/activemodel.yml

```diff
@@ -117,6 +117,8 @@ en:
               none_selected: Select one or more employment types or None of the above if not employed
             has_partner:
               inclusion: Select yes if the client has a partner
+            receives_state_benefits:
+              blank: Select yes if your client receives any benefits
```

Comment:
Very minor and feel free to ignore but I think this should say 'gets' seeing as the page heading says 'gets' and not 'receives' - not sure if this content has already been checked though. I am only really mentioning as I know Louise has been changing 'receives' to 'gets' in a lot of places!